ush/extractvars_tools.sh

```diff
@@ -17,7 +17,7 @@
   done
   mapfile -t requestedvar_in_allgrb2file_arr < "${requestedvar_in_allgrb2file}"
   while read -r vari; do
-    if [[ ! ${requestedvar_in_allgrb2file_arr[*]} =~ ${vari} ]] ;then
+    if [[ ! ${requestedvar_in_allgrb2file_arr[*]} =~ "${vari}" ]] ;then
```

Comment:
@WalterKolczynski-NOAA, can this be shellcheck disabled? The only way to make shellnorms happy here is to remove quotes around  `${vari}`. If these quotes are removed, we would have to put a backslash in front of every regex special character (namely parenthesis) in every parmlist file. Since we want every regex special character to be treated as a literal in these parm files, I think it might be ok to put quotes here.

Reply:
As long as the substitution still happens, yes. I was worried it would treat `${vari}` itself as literal.

Reply:
To suppress the warning
```suggestion
    # shellcheck disable=SC2076
    if [[ ! ${requestedvar_in_allgrb2file_arr[*]} =~ "${vari}" ]] ;then
```

Reply:
Thanks @WalterKolczynski-NOAA. Accidentally put the shellcheck disable in the wrong place. I sent a PR to @NeilBarton-NOAA's `gefs_replay_ci` branch that fixes this.